FakeTrello/Models/Card.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class Card
+    {
+        [Key]
+        public int CardId { get; set; }
```

Comment:
Explore how you would link a user (owner) or users (contributors or people who have been tagged) to a Card. You've got all the classes you need already, so think about how you would add those links with new properties in your Card class.